astroquery/casda/core.py

```diff
@@ -1,12 +1,17 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 # 1. standard library imports
-from io import BytesIO
+from io import BytesIO, StringIO
+from six.moves.urllib.parse import unquote
```

Comment:
we don't actively support python2 any more (we still need to cleanup though), so this should not be from `six`